application/templates/documentation.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>SomTimer</title>
```

Comment:
This is so awesome, can't wait to see what it looks like

Reply:
Same, this is such a rad addition to the BE.